eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/InteropAmqpPropertiesTest.java

```diff
@@ -0,0 +1,184 @@
+
+        final EventHubProducerOptions senderOptions = new EventHubProducerOptions().partitionId(PARTITION_ID);
+        final EventHubConsumerOptions receiverOptions = new EventHubConsumerOptions().retry(Retry.getNoRetry());
+        sender = client.createProducer(senderOptions);
+        receiver = client.createConsumer(getConsumerGroupName(), PARTITION_ID, EventPosition.latest(), receiverOptions);
+    }
+
+    @Override
+    protected void afterTest() {
+        logger.asInfo().log("[{}]: Performing test clean-up.", testName.getMethodName());
+
+        if (client != null) {
+            client.close();
+        }
+
+        if (sender != null) {
+            try {
+                sender.close();
+            } catch (IOException e) {
+                logger.asError().log("[{}]: Sender doesn't close properly.", testName.getMethodName(), e);
+            }
+        }
+
+        if (receiver != null) {
+            try {
+                receiver.close();
+            } catch (IOException e) {
+                logger.asError().log("[{}]: Receiver doesn't close properly.", testName.getMethodName(), e);
+            }
+        }
+    }
+
+    /**
+     * Test for interoperable with Direct Proton Amqp messaging
+     */
+    @Ignore
+    @Test
+    public void interoperableWithDirectProtonAmqpMessage() {
+        skipIfNotRecordMode();
+
+        // Arrange
+        final Message originalMessage = Proton.message();
+        final HashMap<String, Object> appProperties = new HashMap<>();
+
+        appProperties.put(APPLICATION_PROPERTY, "value1");
+        final ApplicationProperties applicationProperties = new ApplicationProperties(appProperties);
+        originalMessage.setApplicationProperties(applicationProperties);
+        originalMessage.setMessageId("id1");
+        originalMessage.setUserId("user1".getBytes());
+        originalMessage.setAddress("eventhub1");
+        originalMessage.setSubject("sub");
+        originalMessage.setReplyTo("replyingTo");
+        originalMessage.setExpiryTime(456L);
+        originalMessage.setGroupSequence(5555L);
+        originalMessage.setContentType("events");
+        originalMessage.setContentEncoding("UTF-8");
+        originalMessage.setCorrelationId("corid1");
+        originalMessage.setCreationTime(345L);
+        originalMessage.setGroupId("gid");
+        originalMessage.setReplyToGroupId("replyToGroupId");
+        originalMessage.setMessageAnnotations(new MessageAnnotations(new HashMap<>()));
+        originalMessage.getMessageAnnotations().getValue().put(Symbol.getSymbol(MESSAGE_ANNOTATION), "messageAnnotationValue");
+        originalMessage.setBody(new Data(Binary.create(ByteBuffer.wrap(PAYLOAD.getBytes()))));
+        final EventData msgEvent = new EventData(originalMessage);
+
+        // Act & Assert
+        StepVerifier.create(receiver.receive())
+            .then(() -> sender.send(msgEvent))
```

Comment:
This is a non-blocking call. I don't think you ever subscribe to it, so the call is never made.